packages/editor/src/lib/hooks/useDocumentEvents.ts

```diff
@@ -274,15 +274,6 @@ export function useDocumentEvents() {
 	}, [editor, container, isAppFocused])
 }
 
-const INPUTS = ['input', 'select', 'button', 'textarea']
-
 function areShortcutsDisabled(editor: Editor) {
```

Comment:
We also seem to have `areShortcutsDisabled` function duplicated a couple of times, could maybe be unified.